mobify/progressive-web-scaffold
# Feedback: Move PW Component Styles

Feedback we received during the Partner Training was that having both the local components and SDK component themes in the same location was confusing. It was suggested that we move the PW component styles to somewhere different to help clarify the difference.

## Changes
- Move all PW-SDK component styles to `app/styles/themes/pw` directory
- Group the node module base style and local theme style imports together in pairs for each component, instead of in two separate groups.

**Before**

```
/app
    /components
        /[local-component]
            _base.scss
        /[pw-component]
            _theme.scss
```

**After**

```
/app
    /components
        /[local-component]
            _base.scss
    /styles
        /themes
            _pw-components.scss
            /pw-components
                _[pw-component].scss
```

**Rationale*

My thought here is to not create a PW directory for just components at the `/app` or `app/components/` level, and leverage the  scaffold's theme folder structure instead to style the PW-SDK components. This folder structure is more or less identical of the project had some other form of theming.

## How to test-drive this PR
- run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=http%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Confirm that all the progressive web component styles work as intended still (nothing broke)
- Consider whether this folder structure is a good idea or not (speak up if not!)
